### PR TITLE
feat: handle back from next step in wallet setup step

### PR DESCRIPTION
**Pull request by greatertomi**
# Checklist

- [x] JIRA - LW-10089
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

On the registration screen, pressing back should take you to the input screen during create wallet

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

### CODE REVIEW

**Review by danielmain — APPROVED** [review]
LGTM ;)

File: apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/components/WalletSetupWizard.tsx [hunk 1]
```diff
@@ -284,7 +286,10 @@ export const WalletSetupWizard = ({
       )}
       {currentStep === WalletSetupSteps.Register && (
         <WalletSetupNamePasswordStepRevamp
-          onBack={moveBack}
+          onBack={() => {
+            setIsBackFromPasswordStep(true);
```
Comment (shawnbusuttil) [inline apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/components/WalletSetupWizard.tsx:290]: I don't mean to nitpick but is it possible to name this differently in a way that doesn't depend on which step comes after?

Comment (greatertomi) [reply apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/components/WalletSetupWizard.tsx:290]: Something like `setIsBackFromNextStep`?

Comment (greatertomi) [reply apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/components/WalletSetupWizard.tsx]: Updated variable on 60fde58. 
IMHO `isBackFromNextStep` is more descriptive.

**Review by oldGreg5 — APPROVED** [review]

**Pull request merged**